lib/index.js

```diff
@@ -126,18 +155,20 @@ Ramen.prototype.identify = function(identify){
 
   // For tracking purposes
   ramenSettings._partner = 'segment.com';
+  ramenSettings.disable_location_watch = true;
 
   // Expose ramenSettings so Ramen.go() can see it
   window.ramenSettings = ramenSettings;
 
   // Ramen.go() will figure things out if called multiple times
   window.Ramen.go();
+
+  this.identified = true;
 };
 
 Ramen.prototype.group = function(group) {
-  // If there is not already a ramenSettings object, or if Ramen isn't loaded,
-  // then do nothing here.
-  if (!(window.ramenSettings && window.Ramen)) {
+  // If there has not been an identify call yet, do nothing.
+  if (!this.identified) {
```

Comment:
Ah, my bad—I'd assumed it would have an `email()` in the same way it has `id()`, like most of our stuff—you'll need to do `this.analytics.user().traits().email` instead.